src/sentry/testutils/cases.py

```diff
@@ -1232,6 +1235,106 @@ def _send_buckets(cls, buckets, entity):
         )
 
 
+class BaseMetricsLayerTestCase(ABC, BaseMetricsTestCase):
+    @abstractmethod
+    def now(self):
```

Comment:
I think this is redundant, the tests will complain anyways if no `now` method is defined

Reply:
Yes but I think that it is better to be explicit than expect the runtime to fail